rwkv/rwkv_tokenizer.py

```diff
@@ -0,0 +1,189 @@
+########################################################################################################
+# The RWKV Language Model - https://github.com/BlinkDL/RWKV-LM
+########################################################################################################
+
+import os, sys, time, random
+
+########################################################################################################
+# Tokenizer #1 (reference, naive, slow)
+########################################################################################################
+
+class RWKV_TOKENIZER():
```

Comment:
I suggest removing reference implementation, since it is not used and (looks like) not intended to be used in `rwkv.cpp`. If anyone wants to take a look at ref impl, they can go to original repository.